### PR TITLE
Allow integration tests to feed stdin to Drush commands.

### PR DESCRIPTION
**Pull request by greg-1-anderson**
One more thing that's now easy to do in the tests.

### CODE REVIEW

File: tests/unish/UnishIntegrationTestCase.php [hunk 1]
```diff
@@ -99,6 +104,13 @@ public function drush($command, array $args = [], array $options = [], $expected
         return $return;
     }
 
+    protected function setStdin($contents)
+    {
+        $path = $this->writeToTmpFile($contents);
```
Comment (weitzman) [inline tests/unish/UnishIntegrationTestCase.php]: If you write to `getSandbox()`, then cleanDirs() will clean up automatically. Would remove all changes in UnishTestCase ... Just a thought.

Comment (greg-1-anderson) [reply tests/unish/UnishIntegrationTestCase.php]: Yeah I like to run locally with UNISH_DIRTY, and I still want these to go away. Maybe segregate the sandbox into invariant-deletion and only-if-not-dirty deletion would work

Comment (weitzman) [reply tests/unish/UnishIntegrationTestCase.php]: Sure, an alwaysDelete subdir of the sandbox sounds good

**Comment by greg-1-anderson** [comment]
Better?

**Pull request merged**